lib/routes/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+const { getHost } = require('../util/getHost');
+
+/* GET home page. */
+router.get('/', function (req, res) {
+  res.render('index.html', {
+    title: 'Welcome',
+    host: getHost(req)
```

Comment:
I don't see a `title` nor a `host` parameter in the index.html template.

In looking through some of your other templates this is continuing to be the case.  I am assuming that they are special parameters for the rendering engine to set the page title and something in regards to the host?

Reply:
They are used by `header.html`, a template which is prepended to all of the page templates.